packages/e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,45 @@
+const { test, expect } = require("@playwright/test");
+
+const { createRandomBankSubAccount } = require("../utils/account");
+
+const { describe, beforeAll, afterAll } = test;
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+
+    beforeAll(async () => {
+        testAccount = await createRandomBankSubAccount();
+    });
+    
+    afterAll(async () => {
+        testAccount && (await testAccount.delete());
+    });
+
+    test("navigates to seed phrase page successfully", async ({ page }) => {
+        await page.goto("/");
+
+        await page.click(
+            `button:text-matches("Import Existing Account", "i")`
+        );
+        await page.click(`data-test-id=recoverAccountWithPassphraseButton`);
+
+        expect(page).toMatchURL(/\/recover-seed-phrase$/);
```

Comment:
When using playwright matchers, many of them are necessarily `async`, so best practice is to _always_ `await` the `expect()` method itself when using playwright matchers.

Since `await`ing something that is sync still works, this is safe even if a particular matcher works synchronously.